### PR TITLE
Fix forceinactive sync (HPC-7506)

### PR DESCRIPTION
**Pull request by itkovian**

### CODE REVIEW

**Comment by itkovian** [comment]
nvm, dates are fixed in other PR :) was looking at the wrong branch. Thx @stdweird

File: test/user.py [hunk 1]
```diff
@@ -102,6 +102,28 @@
     u'force_active': False,
 }
 
+test_account_4 = {
```
[the remark this answers is not in the document]
Comment (itkovian) [reply test/user.py]: how does what work?

[the remark this answers is not in the document]
Comment (stdweird) [reply test/user.py]: adding a variable, what does it change? i looked at the test code, i don't think it gets picked up automatically

[the remark this answers is not in the document]
Comment (itkovian) [reply test/user.py]: It's a new user. With a `forceinactive` status. Used here:

~~~~
        test_account = mkVscAccount(test_account_4)
~~~~

[the remark this answers is not in the document]
Comment (stdweird) [reply test/user.py]: hmm, for some reason i missed the diff of the test/ldapsync.
nvm...

**Pull request merged**